testing/php7-brotli/APKBUILD

```diff
@@ -24,15 +26,15 @@ build() {
 }
 
 check() {
-	cd "$builddir"
+	export SKIP_ONLINE_TESTS=1
 	make NO_INTERACTION=1 REPORT_EXIT_STATUS=1 test
```

Comment:
why not just `make NO_INTERACTION=1 REPORT_EXIT_STATUS=1 SKIP_ONLINE_TESTS=1 test` as patch already merged upstream?